package.json

```diff
@@ -210,5 +211,42 @@
     "sloc": "0.2.0",
     "webpack-dev-server": "^2.4.1",
     "webpack-vendor-chunk-plugin": "1.0.0"
+  },
```

Comment:
@abijames @nk1g09 what is the section doing exactly? I noticed it creates a "pancake" folder in the root, do we need this?

I'm not sure how this will be deployed once the platform is build, specifically if the theming will be correct when deployed. Could we have a screenshot for reference ?